everviz/plugins/_crossplot_plugin/_dash_components/_crossplot_callback.py

```diff
@@ -0,0 +1,17 @@
+def _crossplot_callback(
```

Comment:
Is this the `crossplot_callback` though? This is more just setting the data structure that a plotter accepts. I would say the `update_graph` really is the callback function.

Reply:
Yes, testing the callback is a bit more complicated though, and there really isn't that much logic there, the bulk of the work done in the callback is done here, will be happy to rename if you disagree though.